Spec/PushActivationStateMachine.swift

```diff
@@ -39,27 +37,37 @@ override class var defaultTestSuite : XCTestSuite {
             initialStateMachine = ARTPushActivationStateMachine(rest: rest.internal, delegate: StateMachineDelegate())
         }
 
-        afterEach {
+        func afterEach() {
            rest.internal.resetDeviceSingleton()
         }
 
-        describe("Activation state machine") {
+        
+
+            func test__002__Activation_state_machine__should_set_NotActivated_state_as_current_state_when_disk_is_empty() {
+beforeEach()
 
-            it("should set NotActivated state as current state when disk is empty") {
                 expect(initialStateMachine.current).to(beAKindOf(ARTPushActivationStateNotActivated.self))
+afterEach()
+
             }
 
-            it("should read the current state from disk") {
+            func test__003__Activation_state_machine__should_read_the_current_state_from_disk() {
+beforeEach()
+
                 let storage = MockDeviceStorage(startWith: ARTPushActivationStateWaitingForDeviceRegistration(machine: initialStateMachine))
                 rest.internal.storage = storage
                 let stateMachine = ARTPushActivationStateMachine(rest: rest.internal, delegate: StateMachineDelegate())
                 expect(stateMachine.current).to(beAKindOf(ARTPushActivationStateWaitingForDeviceRegistration.self))
                 expect(storage.keysRead).to(haveCount(2))
                 expect(storage.keysRead.filter({ $0.hasSuffix("CurrentState") })).to(haveCount(1))
                 expect(storage.keysWritten).to(beEmpty())
+afterEach()
```

Comment:
(Observation, doesn't block review)

I'll update the migrator to put the newline _after_ `afterEach`, not before. We'd like this breathing space and SwiftFormat won't help us here.

Reply:
Updated the migrator.